python/sglang/srt/model_executor/model_runner.py

```diff
@@ -386,6 +384,10 @@ def load_model(self):
             load_format=self.server_args.load_format,
             download_dir=self.server_args.download_dir,
         )
+        if self.device == "cpu":
+            self.model_config = update_config(
+                self.model_config, self.load_config, self.tp_size
```

Comment:
`self.load_config` is needed if we want to get the weight_block_size when padding moe, so we move this function to be after the line `self.load_config` is defined.